libs/domain/search/box/box.component.ts

```diff
@@ -29,21 +32,17 @@ import { LitElement, TemplateResult } from 'lit';
 import { query } from 'lit/decorators.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { html } from 'lit/static-html.js';
-import { of } from 'rxjs';
+import { BehaviorSubject, switchMap } from 'rxjs';
 import { SearchBoxOptions, SearchBoxProperties } from './box.model';
 import { baseStyles, searchBoxStyles } from './styles';
 
 @defaultOptions({
   minChars: 2,
-  completionsCount: 5,
-  productsCount: 5,
-  categoriesCount: 5,
-  cmsCount: 5,
-  entries: [
-    SuggestionField.Completion,
-    SuggestionField.Categories,
-    SuggestionField.Products,
-  ],
+  [`${SuggestionField.Suggestions}Count`]: 5,
+  [`${SuggestionField.Products}Count`]: 5,
+  [`${SuggestionField.Categories}Count`]: 5,
+  [`${SuggestionField.Articles}Count`]: 5,
+  entities: defaultEntities,
```

Comment:
what about using the count as an indicator that we need to take the entity into account? Then we can drop the list of entities. Or we introduce a more complex object:

```
@defaultOptions({
  minChars: 2,
  product: {
    max: 5
  }
});
```

With the later, we can also come up with more complex strategies, i.e. to not search on mobile for some content as it's often too busy on a small device.

Reply:
The idea with this one was to avoid `entities: defaultEntities`, as the existence of the object already indicates that the entity is required.